pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -369,3 +369,10 @@ var RetryBackoffReset = settings.RegisterDurationSettingWithExplicitUnit(
 	10*time.Minute, /* defaultValue */
 	settings.DurationInRange(1*time.Second, 1*time.Hour),
 )
+
+var FeatureChangefeedDuplicateCheckEnabled = settings.RegisterBoolSetting(
```

Comment:
pls rename the var and the setting name to be like the other ones, eg `changefeed.duplicate_check.enabled`